Demo/ViewController.swift

```diff
@@ -0,0 +1,58 @@
+import UIKit
+
+class CollectionCell: UICollectionViewCell {
```

Comment:
Demoing this in Unsplash sounds like a great idea! But we still need to have it here so we can run some unit test or UI tests.